frontend/src/components/Post/Post.css

```diff
@@ -0,0 +1,56 @@
+.post-card {
+  border-radius: 10px;
+  border: 1px solid var(--white-shade-2);
+  width: 500px;
+  height: auto;
+  margin-top: 5rem;
+  margin-left: 33rem;
+  background-color: var(--white-shade-0);
+}
+
+.bold {
```

Comment:
This is a generic style, which could be reused in other places in the app as well. Move this somewhere appropriate.